pages/product.js

```diff
@@ -2,116 +2,124 @@ import React from 'react';
 import Rupee from '../components/Icons/Rupee';
 import Bookmark from '../components/Icons/Bookmark';
 import { withRouter } from 'next/router';
+import fetch from 'isomorphic-unfetch';
+import { server } from '../server/config';
 
-class ProductPage extends React.PureComponent{
-
-  state ={
-    qty: 1,
+class ProductPage extends React.PureComponent {
```

Comment:
Please define JSDoc and propTypes.

Reply:
Okay, I will do that. I am working on branch `product-page-patch-1`. We could merge it with this later. Or should  I make changes on this branch ?? I think the first option would be better.